index.html

```diff
@@ -0,0 +1,204 @@
+                    <a href="">Contact</a>
+                </div>
+           </div>
+            <div class="first-nav-line">
+                <span class="logo-image"><img src="./images/logo.png" alt="logo"></span>
+                <span class="invisible-links"> <a href="#">LATEST</a> <a href="">HARD FORK</a> <a href="">PLUGGED</a> <a href="">README</a> <a href="">GROWTHQUARTERS</a></span>
+                <span><a href=""><i class="fas fa-search fa-right fa-1.5x"></i></a><a href=""><i
+                            class="fas fa-bars fa-1.5x hide-icon"></i></a></span>
+            </div>
+            <div class="second-nav-line">
+                <p>Calling young digital talent in the Netherlands. be recognzed in the T500 <i
+                        class="fas fa-arrow-right"></i></p>
+            </div>
+        </nav>
+    </header>
+    <main>
+        <div class="highlights">
+            <div class="highlight-1 highlight-item">
+                <div class="top-left-image">
+                    <img src="images/tnw.svg" alt="logo">
+                </div>
+                <div class="top-right-time">
+                    <p>Tuesday - January 26, 2021</p>
+                </div>
+                <div class="lower-moon">
+                    Opal cuts of your apps' internet access to help you stop doomscrolling
+                </div>
+                <div class="lower-moon-2">
+                    by Ivan Mehta <i class="fas fa-clock"></i> <i class="fas fa-share-alt"></i>
+                </div>
+            </div>
+            <div class="highlight-container">
+                <div class="highlight-2 highlight-item">
+                    <div class="lower-moon">
+                        Google unveils privacy friendly AI cookie killer
+                    </div>
+                    <div class="lower-moon-2">
+                        by Thomas Macaulay <i class="fas fa-clock"></i> <i class="fas fa-share-alt"></i>
+                    </div>
+                </div>
+                <div class="highlight-3 highlight-item">
+                    <div class="lower-moon">
+                        This Mac app automatically detects your duplicate files.
+                    </div>
+                    <div class="lower-moon-2">
+                        by Ivan Mehta <i class="fas fa-clock"></i> <i class="fas fa-share-alt"></i>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="latest-funding">
+            <div class="funding">
+                <div class="funding-header">
+                    <a href="#">Latest Funding Rounds On Index.Co <i class="fa fa-angle-right" aria-hidden="true"></i></a>
+                </div>
+                <div class="funding-links">
+                    <a href="#">See More</a>
+                    <a href="#">Get updates By Email</a>
+                    <a href="#" class="third-funding-link">Add info</a>
+                </div>
+
```

Comment:
The nav menu (See more, Get updates, etc.) are grouped in a `<nav>` tag, not a `<div>` tag.

Reply:
Thanks. All link groupings have been grouped in <nav> instead of <div> tag